stories/documentation/navigation/table-of-content/table-of-content-basic.stories.ts

```diff
@@ -1,45 +1,27 @@
 import { Meta, Story } from '@storybook/angular';
 
 interface TableOfContentBasicStory {
-	mod: string;
 }
 
 export default {
 	title: 'Documentation/Navigation/TableOfContent/Basic',
-	argTypes: {
-		mod: {
-			options: ['', 'mod-grey'],
-			control: {
-				type: 'radio',
-			}
-		},
-	},
 } as Meta;
 
 function getTemplate(args: TableOfContentBasicStory): string {
-	const classes = [args.mod].filter(Boolean).join(' ');
 	return `
-	<nav class="tableOfContent ${classes}">
-		<a class="tableOfContent-item is-active">Section 1</a>
-		<a class="tableOfContent-item">Section 2</a>
-		<a class="tableOfContent-item">Section 3</a>
-		<a class="tableOfContent-item">Section 4</a>
+	<nav class="tableOfContent">
```

Comment:
It would be cool to add an unordered list around the links.
Should we try to do it now (at the risk of a breaking change) or another time?

Reply:
Let's add it now as it's already a bit breaking.